app/Dockerfile

```diff
@@ -22,6 +22,15 @@ RUN set -ex && \
     apk add bash py3-pip curl && \
     apk add --virtual=build gcc libffi-dev musl-dev openssl-dev python3-dev cargo make curl unzip
 
+###
+### Install Checkov
+###
+RUN set -ex && \
+    pip3 install --upgrade pip && \
```

Comment:
why do we use `pip3` here instead of `pip`?

Reply:
Checkov needs python3 to run 
https://github.com/bridgecrewio/checkov#requirements so that is the reason why I'm using pip3

Reply:
I think `pip` uses python 3 interpreter (it'd be consistent with installation of `az cli` below)

Reply:
ok changed to pip